Ginger/GingerCore/Drivers/JavaDriverLib/GingerJavaAgent/agent/com/amdocs/ginger/XPathHelper.java

```diff
@@ -374,6 +375,28 @@ private int GetNameComponentIndex(Component comp)
 
     	return index;
 	}
+
+	public Component GetComponentByMultipleProperties(Window currentWindow, String multipleProperties)
+	{
+		if(currentWindow == null)
+		{
+			GingerAgent.WriteLog("\n Error: Window is not set.");
+			return null;
+		}
+		Container container =currentWindow;
+		String [] pathNodes=multipleProperties.split("/");
```

Comment:
In case of Multi property locate value will have only one node. So no need to split. Also a loop is not needed. Please update this